Actionsheet.hide and rejected promises

**Ionic-native version** : 1.3.2

**Description**:
Create an action sheet. Then in the same method, setTimeout, and hide the actionsheet after some amount of time.

As soon as you call Actionsheet.hide(), and error is thrown.

```
app.bundle.js:38288 STACKTRACE:BrowserDomAdapter.logError @ app.bundle.js:38288ExceptionHandler.call @ app.bundle.js:1588(anonymous function) @ app.bundle.js:24086schedulerFn @ app.bundle.js:1079SafeSubscriber.__tryOrUnsub @ app.bundle.js:85707SafeSubscriber.next @ app.bundle.js:85656Subscriber._next @ app.bundle.js:85606Subscriber.next @ app.bundle.js:85570Subject._finalNext @ app.bundle.js:85362Subject._next @ app.bundle.js:85354Subject.next @ app.bundle.js:85311EventEmitter.emit @ app.bundle.js:1067onError @ app.bundle.js:34962onHandleError @ app.bundle.js:35163ZoneDelegate.handleError @ zone.js:327Zone.runGuarded @ zone.js:233_loop_1 @ zone.js:487drainMicroTaskQueue @ zone.js:494
app.bundle.js:38288 Error: Uncaught (in promise): -1
    at resolvePromise (zone.js:538)
    at zone.js:515
    at Object.callbackFromNative (cordova.js:293)
    at processMessage (cordova.js:1081)
    at processMessages (cordova.js:1104)
    at ZoneDelegate.invoke (zone.js:323)
    at Object.onInvoke (app.bundle.js:35142)
    at ZoneDelegate.invoke (zone.js:322)
    at Zone.run (zone.js:216)
    at zone.js:571BrowserDomAdapter.logError @ app.bundle.js:38288ExceptionHandler.call @ app.bundle.js:1589(anonymous function) @ app.bundle.js:24086schedulerFn @ app.bundle.js:1079SafeSubscriber.__tryOrUnsub @ app.bundle.js:85707SafeSubscriber.next @ app.bundle.js:85656Subscriber._next @ app.bundle.js:85606Subscriber.next @ app.bundle.js:85570Subject._finalNext @ app.bundle.js:85362Subject._next @ app.bundle.js:85354Subject.next @ app.bundle.js:85311EventEmitter.emit @ app.bundle.js:1067onError @ app.bundle.js:34962onHandleError @ app.bundle.js:35163ZoneDelegate.handleError @ zone.js:327Zone.runGuarded @ zone.js:233_loop_1 @ zone.js:487drainMicroTaskQueue @ zone.js:494
zone.js:461 Unhandled Promise rejection: -1 ; Zone: angular ; Task: Promise.then ; Value: -1consoleError @ zone.js:461_loop_1 @ zone.js:490drainMicroTaskQueue @ zone.js:494
zone.js:463 Error: Uncaught (in promise): -1(…)
```


**Sample Code**: 

```javascript
 showActionSheet() {
    let buttonLabels = ['Share via Facebook', 'Share via Twitter'];
    ActionSheet.show({
      'title': 'What do you want with this image?',
      'buttonLabels': buttonLabels,
      'addCancelButtonWithLabel': 'Cancel',
      'addDestructiveButtonWithLabel': 'Delete'
    }).then(buttonIndex => {

      console.log('Button pressed: ' + buttonLabels[buttonIndex - 1]);
      // return false
    });

    setTimeout(() => {
      console.log('should hide')
      ActionSheet.hide()
    }, 5000)
  }
```

**Platforms affected**: Android

--- a/src/plugins/actionsheet.ts
+++ b/src/plugins/actionsheet.ts
@@ -70,5 +70,5 @@ export class ActionSheet {
    * @returns {Promise} Returns a Promise that resolves when the actionsheet is closed
    */
   @Cordova()
-  static hide(): Promise<any> { return; }
+  static hide(options?: any): Promise<any> { return; }
 }

